src/processing/getFiatRate/getClosestHourPrice/index.ts

```diff
@@ -28,6 +32,9 @@ export async function getClosestHourPrice(
             }
             throw new Error('Unknown Response Type');
         } catch (ex) {
+            if (ex instanceof RateLimitError) {
+                throw ex;
+            }
```

Comment:
Should it be an else here?

Reply:
Well, throw, like return, will stop the function. If the condition is not met, it will go to next line.
So the else is not necessary. But we can put it if you prefer that